lib/src/thumbnail_viewer.dart

```diff
@@ -32,7 +32,8 @@ class ThumbnailViewer extends StatelessWidget {
     double _eachPart = videoDuration / numberOfThumbnails;
 
     List<Uint8List> _byteList = [];
-
+    // the cache of last thumbnail
+    Uint8List _lastBytes;
```

Comment:
This should be `Uint8List? _lastBytes;`; note the `?`. Generates an error in null safety otherwise since not declared.

Reply:
I will change it

Reply:
its update in this pr https://github.com/sbis04/video_trimmer/pull/115